src/include/management.h

```diff
@@ -84,6 +85,11 @@ extern "C" {
 #define MANAGEMENT_STATUS             16
 #define MANAGEMENT_SWITCH_TO          17
 
+#define MANAGEMENT_MASTER_KEY         18
+#define MANAGEMENT_ADD_USER           19
+#define MANAGEMENT_UPDATE_USER        20
+#define MANAGEMENT_REMOVE_USER        21
+#define MANAGEMENT_LIST_USERS         22
```

Comment:
A matter of taste, but isn't it better to rename these as `MANAGEMENT_USER_XXX` such as `MANAGEMENT_USER_ADD`? It would be a nice grouping of related options.